python/ray/data/read_api.py

```diff
@@ -1640,35 +1665,6 @@ def from_torch(
     return from_items(list(dataset))
 
 
-def _df_to_block(df: "pandas.DataFrame") -> Block[ArrowRow]:
```

Comment:
I use these utilities in the `plan_<logical_operator>` methods, so they are moved into the `ray.data._internal_util` file